CONTRIBUTING.md

```diff
@@ -1,4 +1,4 @@
-- [Contributing to OpenSearch-SDK](#contributing-to-opensearch-sdk)
+- [Contributing to OpenSearch-SDK](#contributing-to-opensearch-sdk-java)
```

Comment:
Missed this
```suggestion
- [Contributing to OpenSearch-SDK-JAVA](#contributing-to-opensearch-sdk-java)
```

Reply:
and below in the title also. Change `OpenSearch-SDK` -> `OpenSearch-SDK-JAVA`

Reply:
fixed

Reply:
Nit: do we need to capitalize JAVA? It's not an Acronym.

Reply:
Should be `OpenSearch-SDK-Java`

Reply:
fixed